generate_path_node:
  ros__parameters:
    waypoints_x: [0.0, 2.0, 3.0, 4.0]
    waypoints_y: [0.5, 0.5, 1.5, 1.5]